mcp/motors/ec_motors.c

```diff
@@ -2204,6 +2523,24 @@ int initialize_motors(void)
     memset(PivotMotorData, 0, sizeof(PivotMotorData));
 
     motor_ctl_id =  ph_thread_spawn(motor_control, NULL);
+
+    // set up another thread to handle sending/receiving process data at a
+    // higher priority
+    int sendrecv_ret = osal_thread_create_rt(&ecat_sendrecv_thread, \
+        ECAT_SENDRECV_STACKSIZE * 2, \
+        &motor_send_recv, \
+        NULL);
```

Comment:
Why do we need line continuations here?

Reply:
By increasing the thread stack size, I have decreased this line length (removed `* 2` since I increased the stack size so much) and removed the continuations.